recipes/mariadb-connector-c/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "mariadb-connector-c-{}-src".format(self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "${ZLIB_LIBRARY}",
+                              "${ZLIB_LIBRARIES}")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "IF(OPENSSL_FOUND)",
+                              "IF(OpenSSL_FOUND)")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "SET(SSL_LIBRARIES ${OPENSSL_SSL_LIBRARY} ${OPENSSL_CRYPTO_LIBRARY})",
+                              "SET(SSL_LIBRARIES ${OpenSSL_LIBRARIES})")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "${OPENSSL_INCLUDE_DIR}",
+                              "${OpenSSL_INCLUDE_DIR}")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        if self.settings.os == "Windows":
+            self._cmake.definitions["WITH_MSI"] = False
+            self._cmake.definitions["WITH_SIGNCODE"] = False
+            self._cmake.definitions["WITH_RTC"] = False
+        else:
+            self._cmake.definitions["WITH_MYSQLCOMPAT"] = False
+            self._cmake.definitions["WITH_ICONV"] = self.options.with_iconv
+        self._cmake.definitions["WITH_UNIT_TESTS"] = False
+        self._cmake.definitions["WITH_DYNCOL"] = self.options.dyncol
+        self._cmake.definitions["WITH_EXTERNAL_ZLIB"] = True
+        self._cmake.definitions["WITH_CURL"] = self.options.with_curl
+        self._cmake.definitions["WITH_SSL"] = self.options.with_ssl
+        self._cmake.definitions["INSTALL_BINDIR"] = os.path.join(self.package_folder, "bin")
+        self._cmake.definitions["INSTALL_LIBDIR"] = os.path.join(self.package_folder, "lib")
+        self._cmake.definitions["INSTALL_PLUGINDIR"] = os.path.join(self.package_folder, "lib", "plugin")
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING.LIB", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "symbols"))
+
+    def package_info(self):
+        self.cpp_info.names["pkg_config"] = "libmariadb"
+        self.cpp_info.includedirs = [os.path.join("include", "mariadb")]
```

Comment:
I would append the `mariadb` include directory.
When installing in `/usr` (by the system package manager), it is also possible to do `#include <mariadb/xxx>`.

Reply:
done